src/test/java/uk/gov/hmcts/ccd/data/caseaccess/CaseLinkRepositoryTest.java

```diff
@@ -16,14 +18,16 @@
 
 public class CaseLinkRepositoryTest extends WireMockBaseTest {
 
+    private static final String TEST_ADDRESS_BOOK_CASE = "TestAddressBookCase";
+
     @Inject
     private CaseLinkRepository caseLinkRepository;
 
     private CaseLinkEntity caseLinkEntity;
 
     @Before
```

Comment:
Is there a reason these tests use junit 4?  Shouldn't we be using junit 5?

Reply:
WireMockBaseTest setup uses junit 4 so changing these will also need a change in the base class - probably out of scope for this ticket, as this would also need a change in the subsequent class that extend WireMockBaseTest